### DISCUSSION TITLE
Open req ROM 😅

### DISCUSSION THREAD

**Discussion by ghost** [NFS-Project/Rom-Builder#4] [General]
Mangga yang mau req Rom.  Tapi jgn lupa bantu sawer tipis" ya. Buat beli server 😁

**Comment by VioletBlue90** [comment]
Req buat depis lain bisa?

**Reply by ghost** [reply]
Bisa di atur 😁

**Comment by VioletBlue90** [comment]
Tolong setingin dong mas... G mudeng aku cara setingnya

**Reply by ghost** [reply]
Ente kadang kadang.
